src/pspm_dcm.m

```diff
@@ -358,7 +362,7 @@
   isbSn = valid_subsessions(vs);
   sbSn = subsessions(isbSn, :);
   flanks = pspm_time2index(sbSn(2:3), data{sbSn(1)}{1}.header.sr);
```

Comment:
Just a suggestion, should `data{sbSn(1)}{1}.header.sr` be `sr{sbSn(1)}`?

Reply:
This is not something I have changed; so I'd suggest leaving this for a separate pull request.